JoinColony/colonyCDapp
# Return info about expenditure Release action

## Description

This PR adds a new action type describing expenditure finalization and exposes all actions finalizing an expenditure under `finalizingActions` field (most of the time it will be just a single action). This enables the UI to show details of the action initiator.

block-ingestor PR: https://github.com/JoinColony/block-ingestor/pull/194

## Testing

- Create and finalize an expenditure
- Run the following query to see the linked action:
```gql
query ListExpenditures {
  listExpenditures(filter: {nativeId: {ge: 33}}) {
    items {
      nativeId
      status
      finalizingActions: actions(
        filter: {type: {eq: FINALIZE_EXPENDITURE}, isMotionFinalization: {ne: true}}
      ) {
        items {
          id
          type
          initiatorAddress
          initiatorUser {
            profile {
              displayName
            }
          }
          expenditure {
            nativeId
          }
        }
      }
    }
  }
}
```

Resolves #2086
